modules/drivers/athena/test/metabase/driver/athena_test.clj

```diff
@@ -285,3 +289,102 @@
                  (-> @executed-query
                      :native
                      (update :query #(str/split-lines (driver/prettify-native-form :athena %)))))))))))
+
+;;; Athena version of [[metabase.query-processor-test.date-time-zone-functions-test/datetime-diff-mixed-types-test]]
```

Comment:
Code below is moved from `metabase.query-processor-test.date-time-zone-functions-test`